src/containers/products/image-previews.svelte

```diff
@@ -8,6 +8,13 @@
 
   export let productControl;
   export let selectedColor;
+
+  if(productControl.flatImages.length === 0) {
```

Comment:
Put a space here, please :)
Wonder if there's an entry to ESLint to enforce this layout

Reply:
I don't know why I keep missing this particular one that much.
Yes, there is (https://eslint.org/docs/rules/keyword-spacing), and the default is to check for this. Why it is not reporting anything is beyond me.

Reply:
Or on another look, I may have simply overlooked it previously because the console gets flooded with nginx logs right after the change due to hot reloading.
I added the rule anyways